jclouds-plugin/src/main/resources/jenkins/plugins/jclouds/compute/JCloudsCloud/config.jelly

```diff
@@ -9,10 +9,10 @@
         <f:textbox/>
     </f:entry>
     <f:entry title="Max. No. of Instances" field="instanceCap">
-        <f:textbox/>
+        <f:number clazz="positive-number" min="1" step="1" default="1"/>
```

Comment:
Good point. I would be ok with not specifying a default as it could vary widely across clouds and coming up with a good default could be nearly impossible.